packs/fists-of-the-ruby-phoenix-bestiary/spirit-turtle.json

```diff
@@ -2,11 +2,1437 @@
+                },
+                "duration": {
+                    "value": ""
+                },
+                "hasCounteractCheck": {
+                    "value": null
+                },
+                "level": {
+                    "value": 5
+                },
+                "location": {
+                    "heightenedLevel": 5,
+                    "uses": {
+                        "max": 3,
+                        "value": 3
+                    },
+                    "value": "yNEUcNtNackrwWaH"
+                },
+                "materials": {
+                    "value": ""
+                },
+                "prepared": {
+                    "value": ""
+                },
+                "primarycheck": {
+                    "value": ""
+                },
+                "publication": {
+                    "license": "OGL",
+                    "remaster": false,
+                    "title": "Pathfinder Core Rulebook"
+                },
+                "range": {
+                    "value": "60 feet"
+                },
+                "rules": [],
+                "save": {
+                    "basic": "",
+                    "value": ""
+                },
+                "secondarycasters": {
+                    "value": ""
+                },
+                "secondarycheck": {
+                    "value": ""
+                },
+                "slug": "breath-of-life",
+                "spellType": {
+                    "value": "heal"
+                },
+                "sustained": {
+                    "value": false
+                },
+                "target": {
+                    "value": "the triggering creature"
+                },
+                "time": {
+                    "value": "reaction"
+                },
+                "traditions": {
+                    "value": [
+                        "divine"
+                    ]
+                },
+                "traits": {
+                    "rarity": "common",
+                    "value": [
+                        "healing",
+                        "necromancy",
+                        "vitality"
+                    ]
+                }
+            },
+            "type": "spell"
+        },
+        {
+            "_id": "2tnz4SrOCmIg3BY3",
+            "flags": {
+                "core": {
+                    "sourceId": "Compendium.pf2e.spells-srd.Item.SwUiVavHKMWG7t5K"
+                }
+            },
+            "img": "systems/pf2e/icons/spells/tongues.webp",
+            "name": "Tongues (Constant)",
```

Comment:
Since this doesn't exist in the Compendium any longer, I added Truespeech, renamed it, and fixed the publication, slug, and traits from another copy.

Reply:
Use the new name instead of renaming. We're considering the renames to be errata to those items.